### PR TITLE
Update slather action with new commands

### PR DESCRIPTION
**Pull request by bootstraponline**
Fix https://github.com/fastlane/fastlane/issues/4796

Added
- teamcity
- verbose
- source_files
- decimals

### CODE REVIEW

File: fastlane/lib/fastlane/actions/slather.rb [hunk 1]
```diff
@@ -194,10 +211,20 @@ def self.available_options
                                       env_name: "FL_SLATHER_BINARY_BASENAME",
                                       description: "Basename of the binary file, this should match the name of your bundle excluding its extension (i.e. YourApp [for YourApp.app bundle])",
                                       default_value: false),
-          FastlaneCore::ConfigItem.new(key: :binary_file,
```
Comment (KrauseFx) [inline fastlane/lib/fastlane/actions/slather.rb]: Why did you remove this option

Comment (bootstraponline) [inline fastlane/lib/fastlane/actions/slather.rb]: Fixed. I didn't intend to remove any options.

**Comment by bootstraponline** [comment]
I'll submit a new PR since houndci has a bunch of old comments.
